avm/res/compute/virtual-machine/tests/e2e/linux.defaults/main.test.bicep

```
@@ -0,0 +1,102 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-compute.virtualMachines-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'cvmlindef'
```

Please use `min` instead of `def` here and in all default tests. Reason is we have a suppression group for PSRule filtering some rules on default tests which works on `min` substring. Once changed you'll see for example the tag rule will not complain anymore.
```suggestion
param serviceShort string = 'cvmlinmin'
```

---

@eriqua this is actually a curios one. I guess the static tests needs updating, right? One would think this test should have caught it:

https://github.com/Azure/bicep-registry-modules/blob/c6850984b2f57645cddcd5cbc6aa84ca2e3faeff/avm/utilities/pipelines/staticValidation/compliance/module.tests.ps1#L1185-L1196

The issue seems to be that the test assumes a name like `[\/|\\]defaults\/|[\\]` eithen though it should be `[\/|\\].*defaults[\/|\\].+`